README.md

```diff
@@ -25,8 +25,18 @@ dependencies. The one downside is that you must first install Docker!
 The first time you run Splits I/O with Docker is the best, because you'll have time to get a coffee! Yum! After the
 first run, it will be much quicker (on Linux, docker commands may need to be prefixed with sudo).
 ```sh
+docker-compose build
+```
+Once the build finishes, you need to create the database.
+```sh
+docker-compose run web rake db:crete
```

Comment:
It can't be done in `Dockerfile` because the images are not linked at that point.  The only place it could go would be in `docker-compose.yml` before the migrate statement.  Could do something like `command: bash -c "rm -f /app/tmp/pids/server.pid && bundle exec rails db:create db:migrate && bundle exec rails assets:precompile && bundle exec rails s -p 3000 -b '0.0.0.0'"`